packages/fxa-content-server/grunttasks/l10n-extract.js

```diff
@@ -19,7 +19,7 @@ var messagesOutputPath = path.join(
   'LC_MESSAGES'
 );
 var babelCmd =
-  'npx babel --plugins=@babel/plugin-syntax-dynamic-import,dynamic-import-webpack,@babel/plugin-proposal-class-properties --presets @babel/env app/scripts --out-dir .es5';
+  'npx babel --plugins=@babel/plugin-syntax-dynamic-import,dynamic-import-webpack,@babel/plugin-proposal-class-properties --presets @babel/env app/scripts --out-dir .es5 @babel/preset-react';
```

Comment:
`@babel/preset-react';` needs to go right before `@babel/env`, so it looks like `@babel/preset-react,@babel/env`